extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/options/HttpServerOptionsUtils.java

```diff
@@ -2,6 +2,8 @@
 
 import static io.quarkus.vertx.http.runtime.options.TlsUtils.computeKeyStoreOptions;
 import static io.quarkus.vertx.http.runtime.options.TlsUtils.computeTrustOptions;
+import static io.quarkus.vertx.http.runtime.security.HttpSecurityConfiguration.getHttpServerTlsConfigName;
```

Comment:
I don't believe moving these to the HTTP security classes is correct. They are related to TLS and should be closed to the TLS handling. 
Can these methods be moved here, or into a class that is TLS specific?

Reply:
I have introduced `HttpServerTlsConfig` which now holds TLS client auth and config name for the HTTP server. It is not completely decoupled from HTTP Security configuration because if users can configure TLS client auth and config by configuring MTLS, then we still need to make sure that HTTP Security config is ready, so this class still initialize HTTP Security config if it wasn't initialized already (which can only happen in DEV mode).

Hope it's better.